Building on Windows issue

After building on Windows using the setup from the readme, I get the following output running the test:

```
go test -v build_test.go -tor.verbose
# github.com/cretz/bine/process/embedded/tor-0.3.5
c:/gopath/src/github.com/cretz/bine/process/embedded/tor-0.3.5/../../../../tor-static/tor/src/lib/libtor-compress.a(compress_zstd.o): In function `tor_zstd_get_version_str':
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:92: undefined reference to `ZSTD_versionNumber'
c:/gopath/src/github.com/cretz/bine/process/embedded/tor-0.3.5/../../../../tor-static/tor/src/lib/libtor-compress.a(compress_zstd.o): In function `tor_zstd_can_use_static_apis':
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:127: undefined reference to `ZSTD_versionNumber'
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:127: undefined reference to `ZSTD_versionNumber'
c:/gopath/src/github.com/cretz/bine/process/embedded/tor-0.3.5/../../../../tor-static/tor/src/lib/libtor-compress.a(compress_zstd.o): In function `tor_zstd_compress_new':
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:265: undefined reference to `ZSTD_createCStream'
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:275: undefined reference to `ZSTD_initCStream'
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:277: undefined reference to `ZSTD_isError'
c:/gopath/src/github.com/cretz/bine/process/embedded/tor-0.3.5/../../../../tor-static/tor/src/lib/libtor-compress.a(compress_zstd.o): In function `tor_zstd_can_use_static_apis':
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:127: undefined reference to `ZSTD_versionNumber'
c:/gopath/src/github.com/cretz/bine/process/embedded/tor-0.3.5/../../../../tor-static/tor/src/lib/libtor-compress.a(compress_zstd.o): In function `tor_zstd_state_size_precalc':
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:231: undefined reference to `ZSTD_estimateCStreamSize'
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:236: undefined reference to `ZSTD_estimateDCtxSize'
c:/gopath/src/github.com/cretz/bine/process/embedded/tor-0.3.5/../../../../tor-static/tor/src/lib/libtor-compress.a(compress_zstd.o): In function `tor_zstd_compress_new':
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:285: undefined reference to `ZSTD_createDStream'
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:295: undefined reference to `ZSTD_initDStream'
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:297: undefined reference to `ZSTD_isError'
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:299: undefined reference to `ZSTD_getErrorName'
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:314: undefined reference to `ZSTD_freeDStream'
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:279: undefined reference to `ZSTD_getErrorName'
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:312: undefined reference to `ZSTD_freeCStream'
c:/gopath/src/github.com/cretz/bine/process/embedded/tor-0.3.5/../../../../tor-static/tor/src/lib/libtor-compress.a(compress_zstd.o): In function `tor_zstd_compress_process':
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:386: undefined reference to `ZSTD_isError'
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:367: undefined reference to `ZSTD_decompressStream'
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:445: undefined reference to `ZSTD_isError'
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:423: undefined reference to `ZSTD_endStream'
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:428: undefined reference to `ZSTD_isError'
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:362: undefined reference to `ZSTD_compressStream'
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:394: undefined reference to `ZSTD_flushStream'
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:399: undefined reference to `ZSTD_isError'
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:445: undefined reference to `ZSTD_isError'
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:387: undefined reference to `ZSTD_getErrorName'
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:429: undefined reference to `ZSTD_getErrorName'
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:400: undefined reference to `ZSTD_getErrorName'
c:/gopath/src/github.com/cretz/bine/process/embedded/tor-0.3.5/../../../../tor-static/tor/src/lib/libtor-compress.a(compress_zstd.o): In function `tor_zstd_compress_free_':
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:482: undefined reference to `ZSTD_freeDStream'
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:480: undefined reference to `ZSTD_freeCStream'
c:/gopath/src/github.com/cretz/bine/process/embedded/tor-0.3.5/../../../../tor-static/tor/src/lib/libtor-compress.a(compress_zstd.o): In function `tor_zstd_can_use_static_apis':
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:127: undefined reference to `ZSTD_versionNumber'
c:/gopath/src/github.com/cretz/bine/process/embedded/tor-0.3.5/../../../../tor-static/tor/src/lib/libtor-compress.a(compress_zstd.o): In function `tor_zstd_warn_if_version_mismatched':
C:\gopath\src\github.com\cretz\tor-static\tor/src/lib/compress/compress_zstd.c:522: undefined reference to `ZSTD_versionNumber'
collect2.exe: error: ld returned 1 exit status
FAIL	command-line-arguments [build failed]
Command exited with code 2
```

--- a/build.go
+++ b/build.go
@@ -167,6 +167,9 @@ func build(folder string) error {
 		if runtime.GOOS != "darwin" {
 			torConf = append(torConf, "--enable-static-tor")
 		}
+		if runtime.GOOS == "windows" {
+			torConf = append(torConf, "--disable-zstd")
+		}
 		return runCmds(folder, env, [][]string{
 			{"sh", "-l", "./autogen.sh"},
 			torConf,

